sidechain/top-pool-executor/src/getter_operator.rs

```diff
@@ -121,6 +121,6 @@ where
 		shard: &ShardIdentifier,
 		getter: TrustedOperationOrHash<H256>,
 	) -> Result<Vec<H256>> {
-		Ok(self.top_pool_author.remove_top(vec![getter], *shard, false)?)
+		Ok(self.top_pool_author.remove_top(vec![getter], *shard, true)?)
```

Comment:
Here's the meat: The getters were removed while marked as "Invalid". Now they are "InBlock", which is not true either, but as the current "InBlock" does nothing anyway, this hack should be fine for now. Getters should not be processed in the top pool (https://github.com/integritee-network/worker/issues/916), which is why a bigger clean up would be a waste of time IMO.